stable/rbac-manager/Chart.yaml

```diff
@@ -1,18 +1,18 @@
 name: rbac-manager
-version: 0.2.1
+version: 1.0.0
 appVersion: 0.5.0
 description: A Kubernetes operator that simplifies the management of Role Bindings and Service Accounts.
 keywords:
-- rbac
-- authorization
+  - rbac
```

Comment:
I must have had copy-pasta errors.  Oops

Reply:
oh sorry, this is just the result of my automated vscode yaml formatting, that I may end up disabling because of diffs like this